packages/frontend/src/components/connect-button/account-menu/index.tsx

```diff
@@ -50,17 +42,35 @@ export function AccountMenu({
     className,
     chainId,
     onClose,
-    account,
+    address,
     blockie,
 }: AccountMenuProps) {
     const t = useTranslations("accountMenu");
     const [tab, setTab] = useState(Tab.Activity);
     const [copied, setCopied] = useState(false);
+    const [balance, setBalance] = useState<AdapterBlueprint.GetBalanceResult>();
 
     const rootRef = useRef(null);
     const chainData = useChainData(chainId);
     const chainSupported = useIsChainSupported(chainId);
     const { disconnect } = useDisconnect();
+    const { fetchBalance } = useAppKitBalance();
+    const { isConnected } = useAppKitAccount();
+
+    useEffect(() => {
```

Comment:
Is this still needed?

Reply:
Do you mean the effect for the `fetchBalance`? In that case yes.

Reply:
Ah, to fetch the account balance for the native token.